shelley/chain-and-ledger/executable-spec/test/Rules/TestPool.hs

```diff
@@ -148,6 +148,30 @@ registeredPoolIsAdded env ssts =
       M.lookup hk (tSt ^. stPools . to (\(StakePools x) -> x))
         === Just (ledgerSlot env)
 
+-- | Check that a `RetirePool` certificate properly retires a stake pool.
```

Comment:
Wouldn't it be more precise to say that the pool is marked as being retired ? The removal (from `stpool`) is done in `POOLREAP`